src/docs/monarch-roles/mondo_technical_lead.md

```diff
@@ -0,0 +1,27 @@
+## Monarch role: Mondo technical lead
+
+Main responsibilities:
+
+1. Coordinate the development of the Mondo ingest pipeline
```

Comment:
You might want to give additional, more granular details here. For instance, a link to this repository, as well as a diagram that shows this workflow.

Reply:
@matentzn a quick note for now, this still seems fairly general.